aeneas/src/vst/Verifier.v3

```diff
@@ -197,6 +197,22 @@ class Verifier(compiler: Compiler, prog: Program) {
 			}
 		}
 	}
+	def checkStructCycle(decl: VstStruct){
+		var cv = decl.verifier;
+		if (cv.onstack) {
+			cv.FieldDeclError2(decl.token.range(), "cyclic field declaration detected");
+		} else {
+			cv.onstack = true;
+			for (l = decl.members; l != null; l = l.tail) {
```

Comment:
There is a member map, and the `mtref` should have a string; you shouldn't need to render it.

Reply:
I now uses the string in `mtref`. However, I think looping through all members is necessary since here I'm not searching for a member but checking whether any of the members will result in a cycle.